Isolator.dcm

```diff
@@ -696,6 +696,102 @@ K 4Ch Quad Digital Isolator 25Mbps
 F http://www.ti.com/general/docs/lit/getliterature.tsp?genericPartNumber=iso7342fc&fileType=pdf
 $ENDCMP
 #
+$CMP ISO7760DBQ
+D Default high, High-speed, robust EMC, reinforced, six-channel, digital, isolator, SSOP-16
+K 6 channel digital isolator
+F https://www.ti.com/lit/ds/symlink/iso7763.pdf?ts=1596905577895
```

Comment:
Why did you include the `ts` parameter here?

Reply:
FYI: TI's links to datasheets from component pages include a unix timestamp by default for a couple months or so now.
It's not neccessary and setting the ts back manually doesn't seem to open earlier revisions or anything, so I'd leave it out.

Reply:
URLs gibberish to me most of the time. Learned something new and removed the ts parameter.